platform-sdk/swirlds-state-api/src/main/java/com/swirlds/state/State.java

```diff
@@ -91,6 +91,15 @@ default Hash getHash() {
         throw new UnsupportedOperationException();
     }
 
+    /**
+     * Answers the question if the state is already hashed.
+     *
+     * @return true if the state is already hashed, false otherwise.
+     */
+    default boolean isHashed() {
```

Comment:
@imalygin I think it might be a good idea to update JavaDoc here based on what will be finalized in https://github.com/hiero-ledger/hiero-consensus-node/pull/19773 regarding the same method in the proposed `BinaryState`